docs/resources/security_group.md

```diff
@@ -24,30 +24,39 @@ Next step is to attach [exoscale_security_group_rule](./security_group_rule.md)(
 Please refer to the [examples](https://github.com/exoscale/terraform-provider-exoscale/tree/master/examples/)
 directory for complete configuration examples.
 
+<!-- schema generated by tfplugindocs -->
+## Schema
 
-## Arguments Reference
+### Required
 
-[cidr]: https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notatio
+- `name` (String) ❗ The security group name.
 
-* `name` - (Required) The security group name.
+### Optional
 
-* `description` - A free-form text describing the group.
+- `description` (String) ❗ A free-form text describing the group.
+- `external_sources` (Set of String) A list of external network sources, in [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notatio) notation.
+- `timeouts` (Block, Optional) (see [below for nested schema](#nestedblock--timeouts))
 
-* `external_sources` - A list of external network sources, in [CIDR][cidr] notation.
+### Read-Only
 
+- `id` (String) The ID of this resource.
 
-## Attributes Reference
+<a id="nestedblock--timeouts"></a>
+### Nested Schema for `timeouts`
 
-In addition to the arguments listed above, the following attributes are exported:
+Optional:
 
-* `id` - The security group ID.
+- `create` (String)
+- `delete` (String)
+- `read` (String)
 
+* The symbol ❗ in an attribute indicates that modifying it, will force the creation of a new resource.
 
 ## Import
 
 An existing security group may be imported by `<ID>`:
 
-```console
+```shell
 $ terraform import \
   exoscale_security_group.my_security_group \
   f81d4fae-7dec-11d0-a765-00a0c91e6bf6
```

Comment:
nit-picking: Here there is a surviving "$". In other places, this is removed. It could be nice to remove it everywhere (easier copy-paste).

```suggestion
terraform import \
  exoscale_security_group.my_security_group \
  f81d4fae-7dec-11d0-a765-00a0c91e6bf6
```